public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -254,10 +275,20 @@ const getStyles = (theme: GrafanaTheme2, headerHeight: number) => {
       },
     }),
     sidebarContainer: css({
-      position: 'fixed',
-      height: `calc(100% - ${headerHeight}px)`,
+      // the `Resizeable` component overrides the needed `position` and `height`
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      position: 'fixed !important' as 'fixed',
+      height: `calc(100% - ${headerHeight}px) !important`,
```

Comment:
if you set top: headerHeight and bottom: 0, I don't think you need this height !important override

Reply:
Nice - thank you! Did that in 9a3bba393c1917722a536b3eb6c00369bfb8a408 and also moved the overflow to another div.

Now:
![image](https://github.com/user-attachments/assets/5d240ace-080b-407a-a7c9-b2156b982c1c)

Previous:
![image](https://github.com/user-attachments/assets/88af9279-a300-4f3d-a884-688a15be64d7)